docpages/example_programs/the_basics/editing-objects.md

```diff
@@ -0,0 +1,87 @@
+\page editing-objects Editing Channel and Message
```

Comment:
`editing-objects` needs to change to `editing-channel-and-message`. Also seek to renaming the file to editing-channel-and-message`

Reply:
Of course